metricbeat/metricbeat.yml

```diff
@@ -19,6 +19,24 @@ metricbeat.config.modules:
   # Period on which files under path should be checked for changes
   #reload.period: 10s
 
+#============================== Autodiscover ===================================
+
+# Autodiscover allows you to detect changes in the system and spawn new modules
+# as they happen.
+
+metricbeat.autodiscover:
```

Comment:
I think Autodiscovery doesn't need to be in the short config (which we try to keep minimal), but only in the reference one. Especially since Autodiscovery is going to be in the K8s deployment manifersts, right?

Reply:
:+1: that works for me